src/main/java/nl/tudelft/ti2206/group9/gui/Style.java

```diff
@@ -26,6 +26,12 @@
 import javafx.scene.text.Font;
 import javafx.scene.text.FontWeight;
 import nl.tudelft.ti2206.group9.ShaftEscape;
+import nl.tudelft.ti2206.group9.gui.skins.AndySkin;
```

Comment:
PMD doesn't really like the high amount of imports. This requires a solution, keep it in mind!

<review,organization